pages/api/submissions/index.ts

```diff
@@ -0,0 +1,25 @@
+import forms from 'data/flexibleForms/forms';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { addCase } from 'lib/cases';
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  if (req.method === 'POST') {
+    const { socialCareId, formId } = req.body;
+
+    // TODO: start a new submission here
```

Comment:
Can we put ticket numbers (or something) to indicate how these TODOs will be sorted?

Otherwise I'd suggest this shouldn't be in this PR yet if it's not ready

Reply:
this will be a big piece of work in its own right and will probably bloat this pr. the work doesn't have a number yet